stubs/redis/redis/asyncio/client.pyi

```diff
@@ -169,7 +170,7 @@ PSWorkerThreadExcHandlerT: TypeAlias = PubsubWorkerExceptionHandler | AsyncPubsu
 CommandT: TypeAlias = tuple[tuple[str | bytes, ...], Mapping[str, Any]]
 CommandStackT: TypeAlias = list[CommandT]
 
-class Pipeline(Redis[_StrType], Generic[_StrType]):
+class Pipeline(Generic[_StrType]):
```

Comment:
Not a full review yet, but since `Pipeline` derives from `Redis` at runtime, we should keep that relationship. This unfortunately means that we will need lots of `# type: ignore[override]`. See also `redis.client.Pipeline`.